website/static/miner_stats.js

```diff
@@ -74,16 +74,6 @@ function updateChartData(){
 		for (var wh in statData.history[w]) { }
 		//var wh = statData.history[w][statData.history[w].length - 1];
 		var foundWorker = false;
-		for (var i = 0; i < workerHashrateData.length; i++) {
-			if (workerHashrateData[i].key === worker) {
-				foundWorker = true;
-				if (workerHashrateData[i].values.length >= workerHistoryMax) {
-					workerHashrateData[i].values.shift();
-				}
-				workerHashrateData[i].values.push([statData.history[w][wh].time * 1000, statData.history[w][wh].hashrate]);
-				break;
-			}
-		}
 		if (!foundWorker) {
```

Comment:
Found worker (line 76) = false and at line 87 we do check, I believe we can delete this condition.